src/goto-instrument/cover.cpp

```diff
@@ -1461,6 +1461,31 @@ bool instrument_cover_goals(
       cmdline.isset("no-trivial-tests"));
   }
 
+  if(cmdline.isset("cover-traces-must-terminate"))
+  {
+    // instrument a 'fake' goal in CPROVER_START. This will rephrase
+    // the reachability problem  by asking BMC to provide a solution that
+    // satisfies a goal while getting to the end of the program-under-test.
+    const auto sf_it=
+      goto_functions.function_map.find(goto_functions.entry_point());
+    if(sf_it==goto_functions.function_map.end())
+    {
+      msg.error() << "cover-traces-must-terminate: invalid entry point"
```

Comment:
please also emit the content of `entry_point` in the case of an error to facilitate debugging